### PR TITLE
docs(dotnet): Improve some dotnet docs

### PR DESCRIPTION
**Pull request by kblok**
I found some minor things in the dotnet docs. You will find more info in the comments

### CODE REVIEW

File: docs/src/api/class-browsercontext.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ BrowserContexts provide a way to operate multiple independent browser sessions.
 If a page opens another page, e.g. with a `window.open` call, the popup will belong to the parent page's browser
 context.
 
-Playwright allows creation of "incognito" browser contexts with `browser.newContext()` method. "Incognito" browser
+Playwright allows the creation of "incognito" browser contexts with [`method: Browser.newContext`] method. "Incognito" browser
```
Comment (kblok) [inline docs/src/api/class-browsercontext.md:9]: `browser.newContext` is not being "translated"

image

Comment (yury-s) [inline docs/src/api/class-browsercontext.md:9]: how about 'allows the creation of' => 'allows creating' ?

**Review by yury-s — APPROVED** [review]

File: docs/src/assertions.md [hunk 2]
```diff
@@ -183,7 +183,7 @@ assert html == "<p>Result</p>"
 
 ```csharp
 var html = await page.InnerHTMLAsync("div.result");
-Assert.Equals(html, "<p>Result</p>");
+Assert.Equals("<p>Result</p>", html);
```
Comment (kblok) [inline docs/src/assertions.md:186]: Minor. The `expected` argument should come first.

File: docs/src/api/class-browsercontext.md (hunk 1 above)
Comment (kblok) [reply docs/src/api/class-browsercontext.md]: changed!

**Pull request merged**